index.js

```diff
@@ -13,11 +7,15 @@ const methodOverride = require('method-override')
 const bodyParser = require('body-parser')
 const expressValidator = require('express-validator')
 const saveUser = require('./src/user/save_user')
+const dotenv = require('dotenv')
```

Comment:
dotenv needs to be initialized before other modules, notably `debug`, which is why i put it first in the file. can you move this require back to the top?